documentation/IDTA-01001/modules/ROOT/pages/annex/handling-constraints.adoc

```diff
@@ -7,15 +7,19 @@ https://creativecommons.org/licenses/by/4.0/).
 SPDX-License-Identifier: CC-BY-4.0
 
 Illustrations:
-Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, designed by Publik. Agentur für Kommunikation GmbH
+Plattform Industrie 4.0; Anna Salari, Publik.
+Agentur für Kommunikation GmbH, designed by Publik.
+Agentur für Kommunikation GmbH
```

Comment:
See above

Reply:
Fixed in https://github.com/admin-shell-io/aas-specs/pull/452/commits/eb2845590da58d2a5ee077874855d437c0f39299